internal/metrics/cmd.go

```diff
@@ -35,6 +35,45 @@ func (m Metrics) getGPU() []gpuInfo {
 	return gpus
 }
 
+// helper recursive function for getCPU to populate the cpuInfo struct
```

Comment:
even if this is not an exported functions (and I se that this project doesn’t even have linter CI setup… See how old it is), the format for comments are `// <functionname> does something.` (Note the finale dot here).

I’m sure this code is full of counter example, but do you mean updating to it so that at least the new code follows this pattern?

Reply:
It should be fixed now!